kubernetes/deployment-production.tmpl

```diff
@@ -106,3 +106,16 @@ spec:
   selector:
     matchLabels:
       app: aggregation-caesar
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: aggregation-caesar
+spec:
+  selector:
+    app: aggregation-caesar
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  type: NodePort
```

Comment:
I think we should remove this type arg, https://github.com/zooniverse/operations/issues/542

Reply:
Removed that line under the assumption that there's no other config required to make this change.